### PR TITLE
{data}[foss/2020b] harmony v0.1.0-20210528

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2617.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/c69dc5a4d4e8743e52d093f99618d781 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3571.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e383a586661dddd6239be930a48d05e8 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 14 out of 14 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/d3c272ffc1713185684de5e84fa78c08 for a full test report.

File: easybuild/easyconfigs/h/harmony/harmony-0.1.0-20210528-foss-2020b-R-4.0.3.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'harmony'
+local_commit = '2586afd'
+# see DESCRIPTION to determine version,
+# but also take date of last commit into account (since version isn't always bumped)
+version = '0.1.0-20210528'
```
Comment (verdurin) [inline easybuild/easyconfigs/h/harmony/harmony-0.1.0-20210528-foss-2020b-R-4.0.3.eb:7]: Just to check that the reason the version is going backwards is that the previous value was incorrect?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.4
See https://gist.github.com/0a58b833819a3626ed59192651cb5001 for a full test report.

File: easybuild/easyconfigs/h/harmony/harmony-0.1.0-20210528-foss-2020b-R-4.0.3.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/h/harmony/harmony-0.1.0-20210528-foss-2020b-R-4.0.3.eb:7]: The commit `2586afd` from May 28, 2021 shows version `0.1.0`:

https://github.com/immunogenomics/harmony/blob/2586afd4a25f20debd34f70e9766538e39c77b8f/DESCRIPTION#L3

However the commit `88b1e2a` from Feb 24, 2020 that was used in the previous easyconfig https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/harmony/harmony-1.0.0-20200224-foss-2020a-R-4.0.0.eb shows version `1.0.0`: https://github.com/immunogenomics/harmony/blob/88b1e2a1e95306cd4531eaf89ad98712af05ca99/DESCRIPTION#L4

I think this explains why the version is going backwards.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**